### ISSUE TITLE
Able Payer opening too big - question

### ISSUE THREAD

**Issue by RobiCU** [ableplayer/ableplayer#403]
Hi,
When I add AP in my HTML file and call it, the size of AP in the window is too big. How can I make it smaller? I tried to edit width in ableplayer.css, but it's not enough. Also, when I open the CC, the player make the webpage show a scroll bar because it's bigger than the page. How can I change that?

Thank you.
Robi

**Comment by terrill** [comment]
@RobiCU there are at least two ways to control player width: 

1. Add `width` and `height` attributes to the `<video>` element. 
2. Wrap your `<video>` element in a `<div>` or other container, then use CSS to set the width of the container. Able Player will automatically be sized to fit the container.  If you define the width of the container using %, the player will be more responsive.

**Comment by RobiCU** [comment]
Thank you.
I did. I was able to minimize the black borders around the web object in Storyline (SL). I added boxes to cover the remaining black borders (apparently it’s a known issue with SL).
Thank you for the reply ☺

Robi Calderaro M.A.
Instructional Designer & Technical Developer,
Office of Employee Learning and Development
University of Colorado
1800 Grant Street, Suite 400
Denver, CO 80203
t 303 860 6104
www.cu.edu/employee-services/eld<http://www.cu.edu/employee-services/eld>
[cid:image001.png@01D357ED.3357E3E0]
